src/database/dbutils.go

```diff
@@ -0,0 +1,111 @@
+package database
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	pb "accretional.com/semantifly/proto/accretional.com/semantifly/proto"
+	"github.com/jackc/pgx/v5"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
```

Comment:
let's name this file something like postgres.go since these are all postgres-specific functions AFAICT

Reply:
Yeah they are Postgres specifics.